### PR TITLE
fix(leaderboard): update seo metadata

### PR DESCRIPTION
**Pull request by wescopeland**
Updates the title and description for leaderboard pages.

Examples taken from /leaderboardinfo.php?i=2.

**Before**
Title: "Leaderboard"
Description: "Leaderboard: Green Hill Zone - Act 1 (Sonic the Hedgehog, Mega Drive): "

**After**
Title: "Green Hill Zone - Act 1 (Sonic the Hedgehog, Mega Drive)"
Description: "Compete on the Green Hill Zone - Act 1 leaderboard. Rules: Complete this act in the fastest time!"

### CODE REVIEW

File: public/leaderboardinfo.php [hunk 1]
```diff
@@ -72,13 +72,17 @@ function ExplainLeaderboardTrigger(string $name, string $triggerDef, array $code
     echo "</div>"; // devbox
 }
 
+$openGraphDescription = "Compete on the $lbTitle leaderboard. ";
```
Comment (Jamiras) [inline public/leaderboardinfo.php:75]: This page doesn't actually let players compete. I recommend something similar to the achievements OpenGraph description:
```
$openGraphDescription = ($lbDescription ?? $lbTitle) . ", $numEntries entries - $gameTitle for $consoleName";
```

File: public/leaderboardinfo.php [hunk 2]
```diff
@@ -44,7 +44,7 @@
 $consoleName = $lbData['ConsoleName'];
 $forumTopicID = $lbData['ForumTopicID'];
 
-$pageTitle = "Leaderboard: $lbTitle ($gameTitle)";
+$pageTitle = "$lbTitle ($gameTitle, $consoleName)";
```
Comment (Jamiras) [inline public/leaderboardinfo.php:47]: Recommend mimicking achievement info page:
```
$pageTitle = "$lbTitle in $gameTitle ($consoleName)";
```

Comment (wescopeland) [reply public/leaderboardinfo.php]: Updated to your example in latest.

File: public/leaderboardinfo.php (hunk 1 above)
Comment (wescopeland) [reply public/leaderboardinfo.php]: In latest I've set this to:

```php
$openGraphDescription = ($lbDescription ?? $lbTitle) . ", $numEntries entries.";
```

I omitted the game title and console name so the description isn't duplicative with the title.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**